src/ts/helpers/hours.ts

```diff
@@ -8,16 +8,20 @@ const subtypes = [
   'off-road',
 ];
 
+export function isHour(offer): boolean {
+  if (!offer) return false;
+  return _.includes(subtypes, offer.subtype);
```

Comment:
For now we don't have in offer model props that determines if offer is hourly or daily, we just check if offer subtype is in `[groupon, dual-sport, off-road]`, and if so we show hourly form. IMO we should add `prices` prop in api with `hour`, `day`, `week` and `month` arrays with prices and allow user to choose in `base price` what rental plan he wants.